Sources/OpenAPIRuntime/Conversion/CurrencyExtensions.swift

```diff
@@ -177,12 +177,50 @@ extension Converter {
         try decoder.decode(T.self, from: data)
     }
 
+    func convertURLEncodedFormToCodable<T: Decodable>(
+        _ data: Data
+    ) throws -> T {
+
+        let decoder = URIDecoder(
+            configuration: .init(
+                style: .form,
+                explode: true,
+                spaceEscapingCharacter: .percentEncoded,
+                dateTranscoder: configuration.dateTranscoder
+            )
+        )
+        guard let uriString = String(data: data, encoding: .utf8) else {
+            throw RuntimeError.failedToSerializeCodableData
+        }
+
+        return try decoder.decode(T.self, from: uriString)
+    }
+
     func convertBodyCodableToJSON<T: Encodable>(
         _ value: T
     ) throws -> Data {
         try encoder.encode(value)
     }
 
+    func convertBodyCodableToURLFormData<T: Encodable>(
+        _ value: T
+    ) throws -> Data {
+
+        let encoder = URIEncoder(
+            configuration: .init(
+                style: .form,
+                explode: true,
+                spaceEscapingCharacter: .percentEncoded,
+                dateTranscoder: configuration.dateTranscoder
+            )
+        )
+        guard let data = try encoder.encode(value, forKey: "").data(using: .utf8) else {
```

Comment:
To convert the string to Data, use `Data(string.utf8)` instead, as that's not failable and we don't need to handle the error.